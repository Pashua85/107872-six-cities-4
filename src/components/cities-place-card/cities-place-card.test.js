import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import CitiesPlaceCard from './cities-place-card';
import places from '../../mock/test-offers';

Enzyme.configure({
  adapter: new Adapter()
});

describe(`CititesPlaceCard`, () => {
  it(`should render correctly`, () => {
    const wrapper = shallow(
        <CitiesPlaceCard place={places[1]} onMouseEnter={() => {}} onMouseLeave={() => {}} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
