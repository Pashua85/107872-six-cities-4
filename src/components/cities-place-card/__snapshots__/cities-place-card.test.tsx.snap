// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CititesPlaceCard should render correctly 1`] = `ShallowWrapper {}`;
